src/test/java/core/SlaveTest.java

```diff
@@ -40,6 +41,7 @@
 /**
  * @author Orjan Percy <orjan.percy@sonymobile.com>
  */
+@WithPlugins("command-launcher")
```

Comment:
technical debt from https://github.com/jenkinsci/acceptance-test-harness/blob/e3801a6803342ac57234aa3757004ad978ca2935/src/main/java/org/jenkinsci/test/acceptance/po/DumbSlave.java#L38-L48

all `SlaveController` using `DumbSlave` at least in the default setup require this plugin.

It caused a couple of flakes where the save button couldn't be clicked when it was trying to execute the hack, not strictly related to this PR but added for extra stability.